operator/executor/backup.go

```diff
@@ -123,6 +123,7 @@ func (b *BackupExecutor) startBackup(backupJob *batchv1.Job) error {
 	}
 
 	backupJob.Spec.Template.Spec.Containers[0].Env = b.setupEnvVars()
+	backupJob.Spec.Template.Spec.Containers[0].EnvFrom = b.backup.Spec.Backend.EnvFrom
```

Comment:
You probably want to add this to the other jobs as well, or only backups will be able to use those variables.